spring-cloud-starter-stream-source-cdc-debezium/pom.xml

```diff
@@ -95,19 +95,23 @@
 			<groupId>mysql</groupId>
 			<artifactId>mysql-connector-java</artifactId>
 			<version>${version.mysql.driver}</version>
-			<!--<scope>test</scope>-->
```

Comment:
Why is this not a `test` scope any more?

Reply:
Here i just removed an already commented out scope. It think that it was commented on the first place to replace the mysql-connecter excluded from [debezium-connector-mysql](https://github.com/spring-cloud-stream-app-starters/cdc-debezium/blob/master/spring-cloud-starter-stream-source-cdc-debezium/pom.xml#L45)